Safety/boardfiles/Src/stm32f0xx_it.c

```diff
@@ -66,7 +68,10 @@ void NMI_Handler(void)
 void HardFault_Handler(void)
 {
   /* USER CODE BEGIN HardFault_IRQn 0 */
-
+  // Shut down outputs here
+  __HAL_TIM_MOE_DISABLE(&htim1);
+  __HAL_TIM_MOE_DISABLE(&htim3);
```

Comment:
great! Is there any way we can run a restart() function to reset the MCU possible? Would help us recover if its flying at a relatively high altitude. Try looking into nvic_systemreset() maybe.